src/frontend/utils/vilkår.ts

```diff
@@ -0,0 +1,27 @@
+import { Regelverk, VilkårType } from '../typer/vilkår';
+
+export const tilRegelverkVerdi = (value: string): Regelverk | null => {
```

Comment:
Hvis man bruker enumen som value i selecten så bør man ikke trenge å gjøre dette tror jeg? Isåfall kan man kanskje bare si `x as Regelverk` 🤷